src/mmw/js/src/analyze/views.js

```diff
@@ -681,7 +682,7 @@ var StreamTableView = Marionette.CompositeView.extend({
     templateHelpers: function() {
         var data = lodash(this.collection.toJSON()),
             totalLength = data.pluck('lengthkm').sum(),
-            avgChannelSlope = data.pluck('slopepct').sum(),
+            avgChannelSlope = data.pluck('avgslope').sum(),
```

Comment:
Is simple `sum` the right operation here? I was thinking it may make sense to show an average of the average slopes weighted by what percentage of total stream length a stream order represents.

In other words: if stream order 1 represents 90% of the total stream length within the AOI, its slope percentage would be weighted more heavily than another order representing < 10%. On the other hand, this does conflict a bit with the "Total" label on the summation row, since the lengths would be a total but the slope percentage would be an average.

Reply:
I was wondering about this, too. I left it because "Total" means "sum" to me. I think the length-weighted average sounds more meaningful though... @ajrobbins or @jfrankl any insight?